integration_tests/fixtures/prisonApi/getMDIRegime.json

```diff
@@ -0,0 +1,79 @@
+[
```

Comment:
is this the correct place? (see integration_tests/fixtures/activitiesApi/getPrisonRegime.json)